docs-ui/stories/components/checkbox.stories.js

```diff
@@ -19,52 +20,36 @@ export default {
     disabled: {
       control: {type: 'boolean'},
     },
+    checked: {
+      control: {type: 'boolean'},
+    },
   },
 };
 
 export const Default = props => {
-  const [checked, setChecked] = useState(true);
+  const [{checked}, updateArgs] = useArgs();
 
   return (
     <div>
-      <Checkbox
-        {...props}
-        checked={checked}
-        onChange={e => setChecked(e.target.checked)}
-      />
+      <Checkbox {...props} onChange={e => updateArgs({checked: !!e.target.checked})} />
     </div>
   );
```

Comment:
Make use of storybook's `useArgs` hook to automatically update the `checked` arg in the control panel whenever we click on the checkbox:
https://user-images.githubusercontent.com/44172267/218173071-443e63e4-029f-45eb-a6af-6e0e37030afe.mov

Reply:
Ah, good to know about `useArgs`!

Reply:
I just learned about it today. It's great!